__tests__/app.test.js

```diff
@@ -35,3 +35,27 @@ describe("GET /api", () => {
         expect(response.body).toEqual(endpoints)
     })
 })
+
+describe("GET /api/articles/:article_id", () => {
+    test("status: 200 responds with a single article associated with that article ID",
+    async () => {
+        const response = await request(app).get("/api/articles/1")
+        expect(response.status).toBe(200)
+        expect(response.body).toHaveProperty("article")
+        expect(Array.isArray(response.body.article)).toBe(false)
```

Comment:
You dont need to do these two expect statements because below you are access and using article as if it was an object so these are tested for indirectly.